test/processors/text_processor_test.rb

```diff
@@ -32,6 +40,11 @@ class TextProcessorTest < ActiveSupport::TestCase
 
     it 'should analyze file' do
 
+      if travis?
+        audio_monster.expect(:create_temp_file, Tempfile.new('test'), [String, false])
```

Comment:
Line is too long. [86/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.